src/go/rpk/pkg/cli/cmd/redpanda/start_test.go

```diff
@@ -190,6 +190,74 @@ func TestStartCommand(t *testing.T) {
 			// Check that the generated config is as expected.
 			require.Exactly(st, defaultConf, conf)
 		},
+	}, {
```

Comment:
I think we should test 0 as the node id 

also. Is there any other viper config not correctly passing the right information

Reply:
Good point. I just tested it and it's broken for all values.